mods/notepad-remove-launch-new-app-banner.wh.cpp

```diff
@@ -0,0 +1,70 @@
+// ==WindhawkMod==
+// @id              notepad-remove-launch-new-app-banner
+// @name            Notepad Remove Launch New App Banner
+// @description     Removes the banner in classic notepad that tells you to launch the new app.
+// @version         1.0.0
+// @author          ItsProfessional
+// @github          https://github.com/ItsProfessional
+// @include         C:\Windows\System32\notepad.exe
```

Comment:
How about avoiding the hardcoded Windows path with:
```suggestion
// @include         %SystemRoot%\System32\notepad.exe
```

Reply:
Also, what about `C:\Windows\notepad.exe`?